docs/developer-setup/github-authentication.md

```diff
@@ -0,0 +1,55 @@
+# Authenticating with the Github Packages Registry
+
+Some of the dependencies declared for this project include packages stored on the
+[Github Organisation's Packages Registry]. The [Github workflow files] pass secrets through
+github action input variables which in turn set the relevant `GITHUB_ACTOR` and `GITHUB_TOKEN`
+environment variables referred to within the [Project's Gradle settings]. This approach is less than
+ideal for local development so with this in mind, what can a developer do in order to complete
+their set up?
+
+## The what: property keys
+
+There are two gradle properties used to authenticate with the Github packages registry for local
+development:
+
+- `gpr.user`: The developer's Github account name. Associated with the `GITHUB_ACTOR` environment
+  variable.
+- `gpr.token`: The developer's Personal Access Token (PAT), generated in the
```

Comment:
This is only relevant in private repositories right?
So we could have a link to the docs Sandy wrote about PAT token generation / deletion. 
Particularly https://govukverify.atlassian.net/wiki/x/SIIw5g

Reply:
At the time of making this PR, I didn't have access to the URL you've provided. It's certainly worth adding it to the developer documentation!

Reply:
The token is seemingly required regardless of whether the repository is private